src/main/java/org/folio/marccat/resources/TenantAPI.java

```diff
@@ -25,29 +33,26 @@
   private TenantRefService tenantRefService;
 
   @PostMapping
-  public ResponseEntity<String> create(
-    @RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant,
-    @RequestHeader(Global.OKAPI_URL) String okapiUrl,
-    @RequestHeader(Global.OKAPI_TO_URL) String okapiUrlTo,
-    @RequestBody TenantAttributes attributes
-  ) throws SQLException, IOException {
+  public ResponseEntity<String> create(@RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant,
+      @RequestHeader(Global.OKAPI_URL) String okapiUrl, @RequestHeader(Global.OKAPI_TO_URL) String okapiUrlTo,
+      @RequestBody TenantAttributes attributes) throws SQLException, IOException {
     addHeaders(tenant, okapiUrl, okapiUrlTo);
     tenantService.createTenant(tenant, okapiUrl);
-    if(!okapiUrl.isEmpty())
+    if (!okapiUrl.isEmpty())
       tenantRefService.loadData(attributes, okapiHeaders);
     return new ResponseEntity("Success", CREATED);
```

Comment:
Are we relying on Exceptions to indicate failure?

Reply:
I think @carmentrazza will be able to answer your question better

Reply:
Spring Boot ControllerAdvise management is missing. To keep the common behavior of our API I am forced to locally create the failure response with error code 500. I'll make the change myself in the next PR.